packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -97,6 +97,7 @@ export class ActiveWorkflowRunner implements IWebhookManager {
 		private readonly sharedWorkflowRepository: SharedWorkflowRepository,
 		private readonly multiMainSetup: MultiMainSetup,
 		private readonly activationErrorsService: ActivationErrorsService,
+		private readonly workflowService: WorkflowService,
```

Comment:
this unfortunately creating a circular dependency, because 🍝 code. we need to revert the DI changes I requested 😞